pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -143,7 +143,7 @@ public class OfflineClusterIntegrationTest extends BaseClusterIntegrationTestSet
   private static final String MAX_NUM_MULTI_VALUES_MAP_KEY = "maxNumMultiValuesMap";
   // TODO: This might lead to flaky test, as this disk size is not deterministic
   //       as it depends on the iteration order of a HashSet.
-  private static final int DISK_SIZE_IN_BYTES = 20797128;
+  private static final int DISK_SIZE_IN_BYTES = 20843088;
```

Comment:
Why this change ?

Reply:
so I've added a metadata field in the column metadata which increases the disk size. Also based on the comment here it looks like the size may change depending on the iteration order of the `HashSet`. Let me know if you think I should look into this deeper to compare the difference in size change.

Reply:
This change is no longer needed since we removed the field from the metadata. I've changed it back to the original value.